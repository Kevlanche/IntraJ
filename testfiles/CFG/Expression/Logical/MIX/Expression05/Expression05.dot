digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[26]"; "Opt[27]"; "Exit[28]"}
{ rank = same; "List[15]"; "ParameterDeclaration[18]"; "ParameterDeclaration[22]"; "Block[28]"}
{ rank = same; "Modifiers[19]"; "PrimitiveTypeAccess[21]"; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "List[29]"}
{ rank = same; "List[20]"; "List[24]"; "VarDeclStmt[30]"; "ExprStmt[38]"}
{ rank = same; "Modifiers[31]"; "PrimitiveTypeAccess[33]"; "List[34]"; "AssignSimpleExpr[39]"}
{ rank = same; "List[32]"; "VariableDeclarator[35]"; "VarAccess[40]"; "ConditionalExpr[41]"}
{ rank = same; "List[36]"; "Opt[37]"; "AndLogicalExpr[42]"; "OrLogicalExpr[49]"; "AndLogicalExpr[56]"}
{ rank = same; "GTExpr[43]"; "LTExpr[46]"; "GTExpr[50]"; "LTExpr[53]"; "GTExpr[57]"; "LTExpr[60]"}
{ rank = same; "VarAccess[44]"; "IntegerLiteral[45]"; "VarAccess[47]"; "IntegerLiteral[48]"; "VarAccess[51]"; "IntegerLiteral[52]"; "VarAccess[54]"; "IntegerLiteral[55]"; "VarAccess[58]"; "IntegerLiteral[59]"; "VarAccess[61]"; "IntegerLiteral[62]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Expression05 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[18]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[19]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[21]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[22]" [label="ParameterDeclaration\n < c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[27]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[28]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[29]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[30]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[31]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[33]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[35]" [label="VariableDeclarator\n < boolean r >", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[37]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[38]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[39]" [label="AssignSimpleExpr\n < r >", fillcolor=white   style=filled  ]
"VarAccess[40]" [label="VarAccess\n < r >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ConditionalExpr[41]" [label="ConditionalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[42]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[43]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[44]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[45]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[46]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[47]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[48]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"OrLogicalExpr[49]" [label="OrLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[50]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[51]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[52]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"LTExpr[53]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[54]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[55]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"AndLogicalExpr[56]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[57]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[58]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[59]" [label="IntegerLiteral\n < 2 >", fillcolor=white   style=filled  ]
"LTExpr[60]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[61]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[62]" [label="IntegerLiteral\n < 2 >", fillcolor=white   style=filled  ]
"Exit[28]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"Modifiers[19]" -> "List[20]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "Modifiers[19]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "PrimitiveTypeAccess[21]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[22]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[26]" [style=dashed, color=gray]
"Modifiers[31]" -> "List[32]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "Modifiers[31]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "PrimitiveTypeAccess[33]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "List[36]" [style=dashed, color=gray]
"VariableDeclarator[35]" -> "Opt[37]" [style=dashed, color=gray]
"List[34]" -> "VariableDeclarator[35]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "List[34]" [style=dashed, color=gray]
"List[29]" -> "VarDeclStmt[30]" [style=dashed, color=gray]
"AssignSimpleExpr[39]" -> "VarAccess[40]" [style=dashed, color=gray]
"GTExpr[43]" -> "VarAccess[44]" [style=dashed, color=gray]
"GTExpr[43]" -> "IntegerLiteral[45]" [style=dashed, color=gray]
"AndLogicalExpr[42]" -> "GTExpr[43]" [style=dashed, color=gray]
"LTExpr[46]" -> "VarAccess[47]" [style=dashed, color=gray]
"LTExpr[46]" -> "IntegerLiteral[48]" [style=dashed, color=gray]
"AndLogicalExpr[42]" -> "LTExpr[46]" [style=dashed, color=gray]
"ConditionalExpr[41]" -> "AndLogicalExpr[42]" [style=dashed, color=gray]
"GTExpr[50]" -> "VarAccess[51]" [style=dashed, color=gray]
"GTExpr[50]" -> "IntegerLiteral[52]" [style=dashed, color=gray]
"OrLogicalExpr[49]" -> "GTExpr[50]" [style=dashed, color=gray]
"LTExpr[53]" -> "VarAccess[54]" [style=dashed, color=gray]
"LTExpr[53]" -> "IntegerLiteral[55]" [style=dashed, color=gray]
"OrLogicalExpr[49]" -> "LTExpr[53]" [style=dashed, color=gray]
"ConditionalExpr[41]" -> "OrLogicalExpr[49]" [style=dashed, color=gray]
"GTExpr[57]" -> "VarAccess[58]" [style=dashed, color=gray]
"GTExpr[57]" -> "IntegerLiteral[59]" [style=dashed, color=gray]
"AndLogicalExpr[56]" -> "GTExpr[57]" [style=dashed, color=gray]
"LTExpr[60]" -> "VarAccess[61]" [style=dashed, color=gray]
"LTExpr[60]" -> "IntegerLiteral[62]" [style=dashed, color=gray]
"AndLogicalExpr[56]" -> "LTExpr[60]" [style=dashed, color=gray]
"ConditionalExpr[41]" -> "AndLogicalExpr[56]" [style=dashed, color=gray]
"AssignSimpleExpr[39]" -> "ConditionalExpr[41]" [style=dashed, color=gray]
"ExprStmt[38]" -> "AssignSimpleExpr[39]" [style=dashed, color=gray]
"List[29]" -> "ExprStmt[38]" [style=dashed, color=gray]
"Block[28]" -> "List[29]" [style=dashed, color=gray]
"Opt[27]" -> "Block[28]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[27]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[28]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[28]" -> "AssignSimpleExpr[39]" [color=red, constraint=false]
"AssignSimpleExpr[39]" -> "Exit[28]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[39]" -> "GTExpr[50]" [color=red, constraint=false]
"AssignSimpleExpr[39]" -> "LTExpr[53]" [color=red, constraint=false]
"AssignSimpleExpr[39]" -> "GTExpr[57]" [color=red, constraint=false]
"AssignSimpleExpr[39]" -> "LTExpr[60]" [color=red, constraint=false]
"LTExpr[53]" -> "AssignSimpleExpr[39]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[53]" -> "IntegerLiteral[55]" [color=red, constraint=false]
"IntegerLiteral[55]" -> "LTExpr[53]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[55]" -> "VarAccess[54]" [color=red, constraint=false]
"VarAccess[54]" -> "IntegerLiteral[55]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[54]" -> "GTExpr[50]" [color=red, constraint=false]
"GTExpr[50]" -> "AssignSimpleExpr[39]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[50]" -> "VarAccess[54]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[50]" -> "IntegerLiteral[52]" [color=red, constraint=false]
"IntegerLiteral[52]" -> "GTExpr[50]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[52]" -> "VarAccess[51]" [color=red, constraint=false]
"VarAccess[51]" -> "IntegerLiteral[52]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[51]" -> "LTExpr[46]" [color=red, constraint=false]
"LTExpr[60]" -> "AssignSimpleExpr[39]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[60]" -> "IntegerLiteral[62]" [color=red, constraint=false]
"IntegerLiteral[62]" -> "LTExpr[60]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[62]" -> "VarAccess[61]" [color=red, constraint=false]
"VarAccess[61]" -> "IntegerLiteral[62]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[61]" -> "GTExpr[57]" [color=red, constraint=false]
"GTExpr[57]" -> "VarAccess[61]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[57]" -> "AssignSimpleExpr[39]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[57]" -> "IntegerLiteral[59]" [color=red, constraint=false]
"IntegerLiteral[59]" -> "GTExpr[57]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[59]" -> "VarAccess[58]" [color=red, constraint=false]
"VarAccess[58]" -> "IntegerLiteral[59]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[58]" -> "GTExpr[43]" [color=red, constraint=false]
"VarAccess[58]" -> "LTExpr[46]" [color=red, constraint=false]
"LTExpr[46]" -> "VarAccess[51]" [color=blue, constraint=false, xlabel="T "] 
"LTExpr[46]" -> "VarAccess[58]" [color=blue, constraint=false, xlabel="F "] 
"LTExpr[46]" -> "IntegerLiteral[48]" [color=red, constraint=false]
"IntegerLiteral[48]" -> "LTExpr[46]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[48]" -> "VarAccess[47]" [color=red, constraint=false]
"VarAccess[47]" -> "IntegerLiteral[48]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[47]" -> "GTExpr[43]" [color=red, constraint=false]
"GTExpr[43]" -> "VarAccess[47]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[43]" -> "VarAccess[58]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[43]" -> "IntegerLiteral[45]" [color=red, constraint=false]
"IntegerLiteral[45]" -> "GTExpr[43]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[45]" -> "VarAccess[44]" [color=red, constraint=false]
"VarAccess[44]" -> "IntegerLiteral[45]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[44]" -> "VariableDeclarator[35]" [color=red, constraint=false]
"VariableDeclarator[35]" -> "VarAccess[44]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[35]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VariableDeclarator[35]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[26]" -> "Opt[27]" -> "Exit[28]" [style=invis]
"ParameterDeclaration[18]" -> "ParameterDeclaration[22]" [style=invis]
"Modifiers[19]" -> "PrimitiveTypeAccess[21]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" [style=invis]
"VarDeclStmt[30]" -> "ExprStmt[38]" [style=invis]
"Modifiers[31]" -> "PrimitiveTypeAccess[33]" -> "List[34]" [style=invis]
"List[36]" -> "Opt[37]" [style=invis]
"VarAccess[40]" -> "ConditionalExpr[41]" [style=invis]
"AndLogicalExpr[42]" -> "OrLogicalExpr[49]" -> "AndLogicalExpr[56]" [style=invis]
"GTExpr[43]" -> "LTExpr[46]" [style=invis]
"VarAccess[44]" -> "IntegerLiteral[45]" [style=invis]
"VarAccess[47]" -> "IntegerLiteral[48]" [style=invis]
"GTExpr[50]" -> "LTExpr[53]" [style=invis]
"VarAccess[51]" -> "IntegerLiteral[52]" [style=invis]
"VarAccess[54]" -> "IntegerLiteral[55]" [style=invis]
"GTExpr[57]" -> "LTExpr[60]" [style=invis]
"VarAccess[58]" -> "IntegerLiteral[59]" [style=invis]
"VarAccess[61]" -> "IntegerLiteral[62]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
64", fillcolor=1]
Edges [label="Total Nr. Edges
113", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
22", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
25", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}