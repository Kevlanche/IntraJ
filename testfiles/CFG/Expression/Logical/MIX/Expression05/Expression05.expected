NodeName: Entry[13] Successors: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Successors: [VarAccess[44]]
NodeName: VarAccess[44] Successors: [IntegerLiteral[45]]
NodeName: IntegerLiteral[45] Successors: [GTExpr[43]]
NodeName: GTExpr[43] Successors: [VarAccess[47], VarAccess[58]]
NodeName: VarAccess[47] Successors: [IntegerLiteral[48]]
NodeName: IntegerLiteral[48] Successors: [LTExpr[46]]
NodeName: LTExpr[46] Successors: [VarAccess[51], VarAccess[58]]
NodeName: VarAccess[51] Successors: [IntegerLiteral[52]]
NodeName: IntegerLiteral[52] Successors: [GTExpr[50]]
NodeName: GTExpr[50] Successors: [AssignSimpleExpr[39], VarAccess[54]]
NodeName: AssignSimpleExpr[39] Successors: [Exit[28]]
NodeName: Exit[28] Successors: []
NodeName: VarAccess[54] Successors: [IntegerLiteral[55]]
NodeName: IntegerLiteral[55] Successors: [LTExpr[53]]
NodeName: LTExpr[53] Successors: [AssignSimpleExpr[39]]
NodeName: VarAccess[58] Successors: [IntegerLiteral[59]]
NodeName: IntegerLiteral[59] Successors: [GTExpr[57]]
NodeName: GTExpr[57] Successors: [VarAccess[61], AssignSimpleExpr[39]]
NodeName: VarAccess[61] Successors: [IntegerLiteral[62]]
NodeName: IntegerLiteral[62] Successors: [LTExpr[60]]
NodeName: LTExpr[60] Successors: [AssignSimpleExpr[39]]

NodeName: Exit[28] Predecessor: [AssignSimpleExpr[39]]
NodeName: AssignSimpleExpr[39] Predecessor: [GTExpr[50], LTExpr[53], GTExpr[57], LTExpr[60]]
NodeName: GTExpr[50] Predecessor: [IntegerLiteral[52]]
NodeName: IntegerLiteral[52] Predecessor: [VarAccess[51]]
NodeName: VarAccess[51] Predecessor: [LTExpr[46]]
NodeName: LTExpr[46] Predecessor: [IntegerLiteral[48]]
NodeName: IntegerLiteral[48] Predecessor: [VarAccess[47]]
NodeName: VarAccess[47] Predecessor: [GTExpr[43]]
NodeName: GTExpr[43] Predecessor: [IntegerLiteral[45]]
NodeName: IntegerLiteral[45] Predecessor: [VarAccess[44]]
NodeName: VarAccess[44] Predecessor: [VariableDeclarator[35]]
NodeName: VariableDeclarator[35] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: LTExpr[53] Predecessor: [IntegerLiteral[55]]
NodeName: IntegerLiteral[55] Predecessor: [VarAccess[54]]
NodeName: VarAccess[54] Predecessor: [GTExpr[50]]
NodeName: GTExpr[57] Predecessor: [IntegerLiteral[59]]
NodeName: IntegerLiteral[59] Predecessor: [VarAccess[58]]
NodeName: VarAccess[58] Predecessor: [GTExpr[43], LTExpr[46]]
NodeName: LTExpr[60] Predecessor: [IntegerLiteral[62]]
NodeName: IntegerLiteral[62] Predecessor: [VarAccess[61]]
NodeName: VarAccess[61] Predecessor: [GTExpr[57]]
